content/target_mode.md

```diff
@@ -0,0 +1,406 @@
++++
+title = "Target Mode"
+draft = false
+gh_repo = "chef-web-docs"
+
+[menu]
+  [menu.infra]
+    title = "Target Mode"
+    identifier = "chef_infra/features/Target Mode"
+    parent = "chef_infra/features"
+    weight = 80
++++
+
+Target Mode executes Chef Infra Client runs on nodes that don't have Chef Infra Client installed on them.
+
+The target node can be any remote system, edge device, or cloud resource that the host can reach. This includes edge devices, Wi-Fi routers, switches, relays, cloud resources, IP phones, router hubs, and network management peripherals.
+
+## Transport Interface (Train)
+
+Target Mode uses [Transport Interface (Train)](https://github.com/inspec/train) to connect to nodes and execute Chef Infra Client runs.
+
+Target Mode supports the SSH Train protocol. The other Train protocols are experimental.
+
+## Requirements
+
+Target Mode has the following requirements:
+
+- A network-enabled system to execute Target Mode.
+- The `chef-client` CLI. This is included with Chef Workstation.
+- A credentials file which provides the system with information to connect to a target node.
+- A recipe that only includes Target Mode-enabled resources.
+
+## Credentials file
+
+The credentials file defines the connection settings for each node in TOML format.
+
+The credentials file is located in `.chef/credentials` on Linux and Mac systems, or `c:\Users\<USERNAME>\.chef/credentials` on Windows.
+
+### Examples
+
+Define the list of nodes in the credentials file using the TOML format.
+The connection settings for each node are defined using a [TOML Inline Table](https://toml.io/en/v1.0.0#inline-table).
+
+For example, this adds credentials for three nodes using SSH:
+
+```toml
+['HOST-1']
+host = 'target.system.host.1.com'
+user = 'root'
+key_files = '~/.keys/key-pair.pem'
+
+['HOST-2']
+host = 'target.system.host.2.com'
+user = 'root'
+password = '123456'
+
+['192.168.0.252']
+host = '192.168.0.252'
+user = 'root'
+password = '123456'
+
+protocol = ssh
```

Comment:
Another thing which I want to keep is as note here that protocol keyword does not work if using winrm as transport protocol. What works for both ssh and winrm is `transport_protocol`

Reply:
As per Thomas H windows will not be supported for target mode. We will have to make that change later .  You can make the change mentioned above though. @tpowell-progress